src/logics/LALogic.cc

```diff
@@ -775,18 +804,18 @@ void SimplifyConstTimes::constSimplify(const SymRef& s, const vec<PTRef>& terms,
         }
         if (!l.isNumOne(terms[i]))
         {
-            if(l.isNumPlus(terms[i]))
+            if (l.isNumPlus(terms[i]))
                 plus = terms[i];
-            else if(l.isConstant(terms[i]))
+            else if (l.isConstant(terms[i]) and not l.isIte(terms[i]))
```

Comment:
We saw this before somewhere, no? It cannot happen that a PTRef is a constant and an ITE at the same time, no? So the second part of the condition is redundant.